### PR TITLE
Add bsdtar and tidying up packages

### PR DESCRIPTION
**Pull request by kimh**
circleci/circle uses bsdtar for deployment

### CODE REVIEW

**Comment by notnoopci** [comment]
lgtm

Many of these tools aren't truely required for the build, so I may suggest putting them in a `miscellaneous` module

**Comment by kimh** [comment]
I don't think we need to preinstall neither `sysadmin` or `devtools` packages for enterprise and dotcom image, but what do you think?

File: circleci-provision-scripts/base.sh [hunk 1]
```diff
@@ -47,10 +47,25 @@ EOF
...
+lzop
+make
+mercurial
+parallel
+software-properties-common
+tmux
+unzip
+vim
+zip
+EOS
```
Comment (notnoopci) [inline circleci-provision-scripts/base.sh]: `tmux`, `lsof` belong in the devtools as well.  I would move `gfortran` and `liblapack-dev` to its their own functions - Don't think we have many fortran projects :).

I'm torn about `build-essential`, `autoconf`, `cmake`, `make` .  They are primarily relevant to C projects - but other subsequent modules (e.g. ruby, php).  I probably would remove it from base.sh to its own module (e.g. base compilers).

**Comment by notnoopci** [comment]
I think we should install devtools by default as they are quite handy for ssh builds, e.g. for debugging memory/cpu usage of tests (e.g. `htop`) and trying out different tweaks by editting files.

File: circleci-provision-scripts/base.sh (hunk 1 above)
Comment (kimh) [inline circleci-provision-scripts/base.sh]: `gfortran` was needed to build `R` [last time I looked at](https://github.com/circleci/image-maker/pull/298#issuecomment-163943979). I'm not sure how many users actually use `R` but Spotify was using and we need to install anyway. So, let me keep it there.

`liblapack-dev` is required to build `SciPy` which is pretty popular and probably needed for other math library so also want to keep in `base` module.

**Comment by kimh** [comment]
I believe I took care most of feedbacks. Merging now.

**Pull request merged**